lib/components/narrative/narrative-itineraries.js

```diff
@@ -258,9 +264,8 @@ class NarrativeItineraries extends Component {
               )
             ) {
               duplicateItin.allStartTimes.push({
-                arrival: cur.endTime,
-                realtime: firstTransitLegIsRealtime(cur),
-                time: cur.startTime
```

Comment:
The deletion of the `time` field breaks the logic in the above `if` block and is causing for [this search](http://localhost:9966/#/?ui_activeSearch=8enutr3w8&ui_activeItinerary=-1&fromPlace=255%20Devin%20Place%20NE%2C%20Atlanta%2C%20GA%2C%20USA%3A%3A33.81794822741499%2C-84.37911987304688&toPlace=Plaza%20Fiesta%2C%20Buford%20Hwy%20NE%2C%20Atlanta%3A%3A33.86366%2C-84.30758&date=2022-09-21&time=13%3A42&arriveBy=false&mode=&showIntermediateStops=true&ignoreRealtimeUpdates=true&numItineraries=2&otherThanPreferredRoutesPenalty=900) a duplicate bike itinerary to be listed twice (e.g. "Leave at 1:42pm and 1:42pm"). This is not occurring on the `dev` branch.

Reply:
Thank you for catching this! The `dev` merge was quite complex. Fixed in a73b83aef956dfb38929a9fd7ccbd7e4010b3c11